### ISSUE TITLE
Error with python2.7

### ISSUE THREAD

**Issue by yurivict** [ethereum/pyrlp#14]
Trying to make a FreeBSD port. Two problems:
- Build error:

```
byte-compiling /usr/ports/textproc/py-rlp/work/stage/usr/local/lib/python2.7/site-packages/rlp/utils_py3.py to utils_py3.pyc
  File "/usr/local/lib/python2.7/site-packages/rlp/utils_py3.py", line 6
    class Atomic(metaclass = abc.ABCMeta):
                           ^
SyntaxError: invalid syntax
```
- pydistutils produces the wrong plist:

===> Checking for items in pkg-plist which are not in STAGEDIR
Error: Missing: %%PYTHON_SITELIBDIR%%/rlp/utils_py3.pyc
Error: Missing: %%PYTHON_SITELIBDIR%%/rlp/utils_py3.pyo

Does the package forget to exclude utils_py3.py when built with python2?

**Comment by mtgran** [comment]
Hi @yurivict , did you solved the problem? I am having the same issue

**Comment by yurivict** [comment]
nope

**Comment by pipermerriam** [comment]
Fixed via #35 (via #26)
